cluster/addons/cluster-monitoring/heapster-rbac.yaml

```diff
@@ -14,7 +14,7 @@ subjects:
   name: heapster
   namespace: kube-system
 ---
-# Heapster's pod_nanny monitors the heapster deployment & its pod(s), and scales
```

Comment:
The original is grammatically correct, the suggestion is not. "monitors" must end with an s either way; making "heapster" possessive is a matter of preference, but if done you have to remove the article "the".